dev/cut.ts

```diff
@@ -0,0 +1,76 @@
+export const completionSpec: Fig.Spec = {
+  name: "cut",
+  description: "cut out selected portions of each line of a file",
+  args: {
+    template: "filepaths",
+    isOptional: true,
+    variadic: true,
+  },
+  subcommands: [
+    {
+      name: ["-b"],
+      description: "byte positions as a comma or - separated list of numbers",
+      args: [
+        {
+          name: "list",
+          description: "specifies byte positions",
+        },
+        {
+          name: "-n",
+          description: "do not split multi-byte characters",
+          isOptional: true,
+        },
+        {
+          name: "file",
+          template: "filepaths",
+          isOptional: true,
+          variadic: true,
+        },
+      ],
+    },
+    {
+      name: ["-c"],
+      description: "column positions as a comma or - separated list of numbers",
```

Comment:
Same as above. Also, you can safely remove the brackets around the same, since you have only one value:
```suggestion
 {
      name: "-c",
      description: "column positions as a comma or - separated list of numbers",
```

Reply:
Updated this to be an option. Updated name to not have [] around it anymore